app/form_builders/mps_form_builder.rb

```diff
@@ -25,7 +25,7 @@ def radio_toggle_with_textarea(attribute, options = {})
 
   def field_without_label(field_type, attribute)
     content_tag :div,
-      class: form_group_classes(attribute),
+      class: form_group_classes(attribute.to_sym),
```

Comment:
Until https://github.com/ministryofjustice/govuk_elements_form_builder/issues/68 is solved, this is required here to avoid unexpected errors.